src/batch/src/task/task_execution.rs

```diff
@@ -439,9 +455,16 @@ impl<C: BatchTaskContext> BatchTaskExecution<C> {
         task_status: TaskStatus,
         state_tx: &mut tokio::sync::mpsc::Sender<TaskInfoResponseResult>,
         err_str: Option<String>,
+        local_exec_failur_tx: Option<&mut tokio::sync::mpsc::Sender<GetDataResponseResult>>,
```

Comment:
I think `state_tx` and `local_exec_failur_tx` doesn't coexist? One for mpp task, and other for local task, if so, it's better to have some abstraction for this?

Reply:
Yes. Good idea.